backend/api/Areas/Leases/Mapping/Lease/SecurityDepositMap.cs

```diff
@@ -10,12 +12,39 @@ public void Register(TypeAdapterConfig config)
         {
             config.NewConfig<Entity.PimsSecurityDeposit, Model.SecurityDepositModel>()
                 .Map(dest => dest.Id, src => src.SecurityDepositId)
-                .Map(dest => dest.RowVersion, src => src.ConcurrencyControlNumber)
-                .Map(dest => dest.SecurityDepositTypeId, src => src.SecurityDepositTypeCode)
-                .Map(dest => dest.SecurityDepositType, src => src.SecurityDepositTypeCodeNavigation.Description)
                 .Map(dest => dest.Description, src => src.Description)
                 .Map(dest => dest.AmountPaid, src => src.AmountPaid)
-                .Map(dest => dest.DepositDate, src => src.DepositDate);
+                .Map(dest => dest.DepositDate, src => src.DepositDate)
+                .Map(dest => dest.DepositType, src => src.SecurityDepositTypeCodeNavigation)
+                .Map(dest => dest.OtherTypeDescription, src => src.OtherDepositTypeDesc)
+                .Map(dest => dest.PersonDepositHolder, src => src.PimsSecurityDepositHolders.FirstOrDefault(h => h.Person != null))
```

Comment:
Do we have business approval on converting this many relationship to a 1-1? If so, do we need some method of sorting here?

Reply:
This is one of those places where the db model does not match the requirements. It looks like the deposits can have multiple holders, however, the requirements imply that there is only one holder.

We will have to revisit the holder logic once we work on the contact manager story.

Reply:
Yes, lets bring this up during backlog refinement tomorrow.